examples/main/constants/toc.js

```diff
@@ -1,15 +1,17 @@
 // Modules
-import MarkerExample from '../views/marker';
-import NotInteractiveExample from '../views/not-interactive';
-import CustomOverlayExample from '../views/custom-overlay';
-import RouteOverlayExample from '../views/route-overlay';
-import StyleDiffingExample from '../views/style-diffing';
-import ClickExample from '../views/click';
+// import MarkerExample from '../views/marker';
```

Comment:
This will actually affect the `main` example when ran locally as well. Can't we just filter out the specific part of the array when rendering these? Or export different chunks of it.